
{'date': 'Wed Jul 15 2020 15:35:50.493 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 19
  cena1.vai()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Jul 15 2020 15:36:12.131 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 19
  cena1.vai()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Jul 15 2020 15:36:36.717 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 20
    teste()    
  module <module> line 18
    	cena1 = Cena(link_fundo)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 15:37:55.655 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 20
    teste()    
  module <module> line 18
    	cena1 = Cena(link_fundo)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 15:42:33.607 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    teste()   
  module <module> line 19
    	cena1 = Cena(link_fundo)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 15:44:54.539 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    teste()   
  module <module> line 20
    	cena1 = Cena(link)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 15:46:48.224 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 16
  link_fundo = "https://img.elo7.com.br/product/original/1AD3471/painel-1x0-70-salao-de-festa-salao-de-festa.jpg
                                                                                                                ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Wed Jul 15 2020 15:47:24.853 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    teste()   
  module <module> line 20
    	cena1 = Cena(link_fundo)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 15:52:14.244 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    teste()   
  module <module> line 20
    	cena1 = Cena(img = link_fundo)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 15:53:34.227 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    teste()   
  module <module> line 20
    	cena = Cena(img = link_fundo)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 15:53:51.56 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    teste()   
  module <module> line 20
    	cena = Cena(img = link)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 15:54:02.381 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    teste()   
  module <module> line 20
    	cena = Cena(img=link)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 15:57:20.309 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    teste()
  module <module> line 20
    	cena=Cena(img=link)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 15:59:06.918 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    teste()
  module <module> line 20
    	cena= Cena(img=link)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 16:00:09.447 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    teste()
  module <module> line 20
    	cdd= Cena(img=CDD)
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Jul 15 2020 16:00:50.866 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''ok
ok pt2
ok pt3
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    teste()
  module <module> line 23
    	cena.vai()
NameError: name 'cena' is not defined
'''},
{'date': 'Wed Jul 15 2020 16:04:43.697 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 17
  def teste()
              ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 15 2020 16:13:55.530 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    teste()
  module <module> line 19
    	cena2 = Cena(img = fundo, style=dict(left=135, top=400, width=20, height="20px"))    
  module _spy.vitollino.main line 1001
    Cena.c(**kwargs)
  module _spy.vitollino.main line 1044
    imagem, kwargs = (imagem, {}) if isinstance(imagem, str) \
KeyError: img
'''},
{'date': 'Wed Jul 15 2020 16:22:23.759 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: Cannot read property 'charAt' of undefined>
'''},
{'date': 'Wed Jul 15 2020 16:22:35.372 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: Cannot read property 'charAt' of undefined>
'''},
{'date': 'Wed Jul 15 2020 16:23:43.836 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 41
    _ = Kamui()
  module <module> line 31
    def pula_issun(_=0, issun_=self.issun.elt.style):
AttributeError: 'Kamui' object has no attribute 'issun'
'''},
{'date': 'Wed Jul 15 2020 16:25:21.669 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    _ = Kamui()
NameError: name 'Kamui' is not defined
'''},